README.md

```diff
@@ -2,6 +2,8 @@
 
 HERE Data SDK for TypeScript is a TypeScript client for the <a href="https://platform.here.com" target="_blank">HERE platform</a>.
 
+To learn how to install and use the Data SDK, see the <a href="https://github.com/heremaps/here-data-sdk-typescript/blob/master/docs/GettingStartedGuide.md" target="_blank">Getting Started Guide</a> and <a href="https://developer.here.com/documentation/sdk-typescript/dev_guide/index.html" target="blank">Developer Guide</a>.
```

Comment:
In Data SDK for C++, you added a separate section for this paragraph. Please align the structure of this page with the Readme structure of Data SDK for C++ where possible.

Reply:
Added